.travis.yml

```diff
@@ -6,11 +6,7 @@ php:
 
 # list of paths/bundle to execute
 env:
-    # run setup wizard to install Demo+Content and then test demo with content
   - INSTALL="demoContentNonUniqueDB" PROFILE="demo" TEST="content"
```

Comment:
If you are stuck on getting demo data to work, then it is ok to comment out this for now. ref mail to @ezsystems/qa-team